src/api/stats/getAmmPrices.ts

```diff
@@ -580,7 +585,19 @@ const updateAmmPrices = async () => {
       };
     };
 
-    const ammPrices = fetchAmmPrices(pools, knownPrices);
+    const ammPrices = fetchAmmPrices(pools, knownPrices).then(prices => {
+      //Set prices for the wrapped version of native tokens (if native was set)
+      const nativeTokens = new Set(
+        Object.values(addressBookByChainId).map(addressbook =>
+          addressbook.tokens.WNATIVE.symbol.slice(1)
+        )
+      );
+      nativeTokens.forEach(nativeToken => {
+        if (prices.tokenPrices.hasOwnProperty(nativeToken))
+          prices.tokenPrices[`W${nativeToken}`] = prices.tokenPrices[nativeToken];
```

Comment:
I'm quite confident that prices should be extracted from wrapped to native, not vice versa. My reasoning is based on opinion that most AMMs are Uniswap v2 clones which operates wrapped tokens. On another hand the only AMM working with native directly 1inch which is not present on all networks. 
Could you please check and post here pairs and their liquidities for data-driven decision. Should be double by console.log somewhere inside `fetchAmmPrices`. Agree?

Reply:
I mean it's symbolic wether we use one or the other, it won't make a difference. Wether we initially set them all as native and later derive the wrapped value or vice versa is the same